src/libs/LangChain.Core/Chains/StackableChains/Agents/Crew/CrewAgent.cs

```diff
@@ -207,10 +149,12 @@ protected override async Task<IChainValues> InternalCall(IChainValues values)
                 }
 
                 var tool = _tools[action.Action];
-                var toolRes = await tool.ToolTask(action.ActionInput).ConfigureAwait(false);
+                var toolRes = await tool.ToolTask(action.ActionInput);
                 ActionResult(toolRes);
```

Comment:
Libraries that do not interact with the UI should use .ConfigureAwait(false) for all async methods. This will not hijack the current thread and speeds up the code.